### DISCUSSION TITLE
Drag'n drop of collections in the Data model settings

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13282] [Feature Requests]
Drag'n drop of collections is a nice feature to reorganize the list hence the menus.
But it is a frustrating experience. So my suggestions:

- When moving the handle and finally dropping the collection, the collection opens itself most of the time: it takes twice the necessary time to order tens of collections. => a click on the handle should never open the collection
- It is very difficult to align vertically sub-collections. They jump too left or too right. => the vertical could be catch more easily.

**Comment by benhaynes** [comment]
I agree on both points! Good recommendations. We can remove the click (to open) event on the drag handle. The indentation part is always tricky, but it'd be great to get that a little easier (eg: bigger hitzones). Would love a PR from the community! ❤️

**Comment by rijkvanzanten** [comment]
This has since been implemented 🙂
